packages/shared/src/components/modals/SquadsBetaModal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement, useState } from 'react';
+import SquadsBackgroundSvg from '../../svg/SquadsBackground';
+import { Button } from '../buttons/Button';
+import { Justify } from '../utilities';
+import { Modal } from './common/Modal';
+import {
+  ConfirmationButtons,
+  ConfirmationDescription,
+  ConfirmationHeading,
+  ConfirmationModal,
+} from './ConfirmationModal';
+
+export type SquadsBetaModalProps = {
+  onRequestClose: () => void;
+  onNext: () => void;
+};
+
+type ConfirmationProps = {
+  onRequestClose: () => void;
+  onContinue: () => void;
+};
+
+function Confirmation({
+  onRequestClose,
+  onContinue,
+}: ConfirmationProps): ReactElement {
+  return (
+    <ConfirmationModal isOpen>
+      <ConfirmationHeading>Quit the process?</ConfirmationHeading>
+      <ConfirmationDescription>
+        <p>
+          Learning is more powerful together. Are you sure you want to quit the
+          process?
+        </p>
+        <p>p.s you can create a new squad from the left sidebar</p>
+      </ConfirmationDescription>
+      <ConfirmationButtons>
+        <Button className="btn-secondary" onClick={onRequestClose}>
+          Cancel
+        </Button>
+        <Button className="btn-primary-ketchup" onClick={onContinue}>
+          Continue
+        </Button>
+      </ConfirmationButtons>
+    </ConfirmationModal>
+  );
+}
```

Comment:
Perhaps we can abstract this into it's own component (or the function you once suggested?)

Reply:
Lets move this to its own file for now, and create a task for the function.

Reply:
Done: https://dailydotdev.atlassian.net/browse/WT-888